### ISSUE TITLE
Zero cloud scroll rate in default EE sky settings

### ISSUE THREAD

**Issue by AtlasLinden** [secondlife/viewer#1355]
### Environment

Second Life Release 7.1.6.8745209917 (64bit)

### Description

See https://github.com/secondlife/viewer/pull/892

### Reproduction steps

Test plan:
- Verify new sky settings have a cloud scroll of (0.2, 0.01) by default
- Verify Midday setting is static

**Comment by AtlasLinden** [comment]
This may also require a server side fix

**Comment by DmitrySProductengine** [comment]
Failed QA.
Verified on the Second Life Release 7.1.7.8974243247 (64bit) on Win10/OSX in the scope of [IQA #214](https://github.com/secondlife/iqa/issues/214).

The midday setting is static for cloud scroll.
But the new sky setting still has a cloud scroll setting of (0, 0) by default.
Also, the value 0.01 is not entered into the cloud scroll fields - the value is rounded to 0 (the allowable value step is 0.05).
See the attached screenshot ‘Actual - 7.1.7.8974243247.png’
<img width="1290" alt="Actual - 7 1 7 8974243247" src="https://github.com/secondlife/viewer/assets/136308794/8a1098c6-133f-4f52-8c42-4230e48a53c2">

**Comment by AtlasLinden** [comment]
Maint X is promoted to default.
Moving this to Maint C for now.

**Comment by dmytrohproductengine** [comment]
Deferred QA.

There are no changes in the viewer, the server side is not done. The ticket status has not changed.

Verified on the Second Life Release 7.1.12.12258017789 (64bit) on Win10/OSX in scope https://github.com/secondlife/iqa/issues/414.

**Comment by Dan-Linden** [comment]
Reopened.
Verified on Second Life Release 26.2.0.24254827122 on Win11/OSX in the scope of https://github.com/secondlife/iqa/issues/635

A New Sky has a Cloud Scroll rate of X:0  Y:0.
Also, as DmitrySProductengine said above, the value 0.01 is not entered into the cloud scroll fields - the value is rounded to 0 (the allowable value step is 0.05)

**Comment by akleshchev** [comment]
I assume server side needs to modify default asset 651510b8-5f4d-8991-1592-e7eeab2a5a06 with new defaults, but I'm not sure if it's actually needed.

But '0.01' is viewer sided, fixed that. Check default day cycle, it should now dysplay small values properly and allow setting them.

**Comment by Dan-Linden** [comment]
Tested on Second Life Release 26.3.0.26586584194 on Win11/OSX in the scope of secondlife/iqa#660

The viewer-side smallest step for clouds is 0.05. Typing in 0.03 will round to 0.05.
Is this expected behavior?

**Comment by akleshchev** [comment]
The change is in develop branch, not in 26.3, moving to 26.4

**Comment by Dan-Linden** [comment]
Tested on Second Life Release 26.4.0.33894281402 on Win11/OSX in the scope of https://github.com/secondlife/iqa/issues/604 

New Sky - the Cloud Scroll is 0, 0
New Day - the Cloud Scroll is <0.5, 0.01> on keyframes 1,2,3,7,8 and <0.2, 0.01> on keyframes 4,5,6.
I can successfully set the Cloud Scroll to small values like -0.01 , 0.01 with this viewer.

The original description of this bug says "Verify new sky settings have a cloud scroll of (0.2, 0.01) by default"
Is that a requirement?

**Comment by akleshchev** [comment]
> The original description of this bug says "Verify new sky settings have a cloud scroll of (0.2, 0.01) by default"

No, I think that is more for https://github.com/secondlife/viewer/pull/892, for this one finer control of the values is enough.

**Comment by Dan-Linden** [comment]
Passed QA.

Does https://github.com/secondlife/viewer/pull/892 need a new issue created?
